nndt/space2/filesource.py

```diff
@@ -6,9 +6,24 @@
 
 
 class FileSource(AbstractBBoxNode, IterAccessMixin):
+    """
```

Comment:
Sklearn uses docstrings for the class https://github.com/scikit-learn/scikit-learn/blob/36958fb24/sklearn/linear_model/_stochastic_gradient.py#L903

Statmodels do the same: https://github.com/statsmodels/statsmodels/blob/main/statsmodels/imputation/mice.py

I think this is more convenient for user.